src/precice/impl/SolverInterfaceImpl.hpp

```diff
@@ -380,15 +406,33 @@ class SolverInterfaceImpl {
       const double *values);
 
   /**
-   * @brief Write vectorial data to the interface mesh
+   * @copydoc precice::SolverInterface::writeBlockVectorGradientData()
+   */
+  void writeBlockVectorGradientData(
+      int           fromDataID,
+      int           size,
+      const int *   valueIndices,
+      const double *values,
+      bool          rowMajor = false);
+
+  /**
+   * @brief Write scalar data to the interface mesh
    *
    * The exact mapping and communication must be specified in XYZ.
    *
-   * @param[in] fromDataID ID of the data to be written, e.g. 1 = forces
-   * @param[in] dataPosition Position (coordinate, e.g.) of data to be written
-   * @param[in] dataValue Value of the data to be written
+   * @param fromDataID       [IN] ID of the data to be written (2 = temperature, e.g.)
+   * @param dataPosition [IN] Position (coordinate, e.g.) of data to be written
+   * @param dataValue    [IN] Value of the data to be written
```

Comment:
If you intend to rearrange the function list, please do not use all caps for `IN`

Reply:
If you intend to rearrange the function list, please do not use all caps for `IN`

Reply:
These weren't my changes, they got applied after i pulled the latest changes from the develop branch, since there were conflicts in this class. Should I revert them ?